app/helpers/logging.py

```diff
@@ -58,10 +57,9 @@ def tick(self):
                 should_log = True
 
         if should_log:
-            self.logger.info(self.desc + " [" + '{:.2f}'.format(round(float(self.current_step) / float(self.total_steps) * 100, 2)) + "% done" + "]")
-
-    def print_section_divider(self):
-        self.logger.info("-" * 50)
+            time_diff = max(float(1), float(dt.datetime.today().timestamp() - self.start_time))  # avoid a division by zero
+            ticks_per_second = "{:,}".format(round(float(self.current_step) / time_diff))
+            self.logger.info(self.desc + " [" + ticks_per_second + " eps. - " + '{:.2f}'.format(round(float(self.current_step) / float(self.total_steps) * 100, 2)) + "% done" + "]")
```

Comment:
General remark, the logger actually has string formatting built in, it results in easier to read code.
E.g.: instead of 
```python
logger.info("The number is {0}".format(my_number))
```
you can use
```python
logger.info("The number is %d", my_number)
```